### PR TITLE
Optimize sample data injector script

### PR DESCRIPTION
**Pull request by snowteamer**
Related to #13 and #14

### CODE REVIEW

File: sampleDBData.js [hunk 1]
```diff
@@ -1,54 +1,96 @@
...
+  return chooseRandomInteger(1, 9999) / 100;
+};
+
+const chooseRandomRating = () => {
+  return chooseRandomInteger(0, 10) / 2;
+};
+
+const createRandomProduct = () => {
+  return new Product({
+    name: 'Product',
```
Comment (magnus195) [inline sampleDBData.js:35]: I got an error here saying that productName is required. We need to make sure that the schema has the right keys

File: sampleDBData.js [hunk 2]
```diff
@@ -1,54 +1,96 @@
...
+    ],
+    quantityType: 'things',
+    averageRating: chooseRandomRating(),
+    reviewCount: chooseRandomInteger(0, 999),
+    category: 'Good products',
+    price: chooseRandomPrice(),
+    brandName: 'Supreme',
+    brandLogoLink:
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/2/28/Supreme_Logo.svg/576px-Supreme_Logo.svg.png',
+    discounts: [
```
Comment (magnus195) [inline sampleDBData.js:49]: Same error with discountName

**Comment by Twaha-Rahman** [comment]
@snowteamer I think this PR will need some changes due to the naming changes

**Comment by snowteamer** [comment]
Yes, I was waiting on #30 to be accepted to rebase this PR. Or should I make the same changes here?

**Comment by Twaha-Rahman** [comment]
@snowteamer Why did you close this PR? Are you going to create a new PR based on the current master branch? Things in this PR was really good!

**Comment by snowteamer** [comment]
I'm confused, I don't remember closing it. 
>  @Twaha-Rahman Twaha-Rahman closed this in #32 1 hour ago 
Did you close it by accident?

**Comment by Twaha-Rahman** [comment]
> In fact I think it was automatically closed when you merged #32, since its summary included
> 
> > Closes #29

Oh... I see! Please continue with the PR.

**Comment by Twaha-Rahman** [comment]
What's the current status of this PR?

**Comment by snowteamer** [comment]
It was designed for a single type of data, namely products; moreover we made quite a lot of changes to this script recently. As you know, it's too early to optimize a script before it's api is stable enough. so I left this PR waiting for now. But I've written a few more little randomization functions in the meantime which I'd like to add to this PR later.

**Pull request closed without merging**